pages/recipes/[id]/index.js

```diff
@@ -49,8 +53,8 @@ export default function DetailsPage() {
       <StyledDetailsItem>
         <h4>Ingredients:</h4>
         <ul>
-          {recipe.ingredients.map((ingredient) => (
-            <li key={ingredient.id}>
+          {recipe.ingredients.map((ingredient, id) => (
+            <li key={id}>
```

Comment:
this works for the moment, however, using the id of the entry within the array is not an ideal solution and should only be used as a last resort.

Since this Pull Request, the ingredients of all future ingredients have the key `ingredientID`.

So going forward, I recommend changing the key to be `ingredient.ingredientID`. However, this will not work for already existing recipes, since they don't have that key.